File: 0682/sesión25 - Reportes/Back-End/ServicioRESTFul/SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/TiposProductosDTO.java
package pe.edu.pucp.softinv.model;

public class TiposProductosDTO {

    private Integer tipoProductoId;
    private String nombre;
    private String descripcion;

    public TiposProductosDTO() {
        this.tipoProductoId = null;
        this.nombre = null;
        this.descripcion = null;
    }
    
    public TiposProductosDTO(Integer tipoProductoId, String nombre, String descripcion) {
        this.tipoProductoId = tipoProductoId;
        this.nombre = nombre;
        this.descripcion = descripcion;
    }

    /**
     * @return the tipoProductoId
     */
    public Integer getTipoProductoId() {
        return tipoProductoId;
    }

    /**
     * @param tipoProductoId the tipoProductoId to set
     */
    public void setTipoProductoId(Integer tipoProductoId) {
        this.tipoProductoId = tipoProductoId;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the descripcion
     */
    public String getDescripcion() {
        return descripcion;
    }

    /**
     * @param descripcion the descripcion to set
     */
    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
    
}

File: 0682/sesión 15  - Master Detail/con Master Detail/SoftInv/SoftInvModel/TiposProductosDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.Model
{
    public class TiposProductosDTO
    {
        private int? tipoProductoId;
        private string nombre;
        private string descripcion;

        public TiposProductosDTO()
        {
            this.TipoProductoId = null;
            this.Nombre = null;
            this.Descripcion = null;
        }

        public TiposProductosDTO(int tipoProductoId, string nombre, string descripcion)
        {
            this.TipoProductoId = tipoProductoId;
            this.Nombre = nombre;
            this.Descripcion = descripcion;
        }

        public int? TipoProductoId { get => tipoProductoId; set => tipoProductoId = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
    }
}
